services/_shared/personas/code_reviewer/diff_parser.py

```
@@ -55,14 +59,17 @@ class DiffHunk:
 
     def __post_init__(self) -> None:
-        assert self.file_path, "DiffHunk.file_path must be non-empty"
-        assert self.new_start >= 1, (
-            f"DiffHunk.new_start must be >= 1 (got {self.new_start}); "
-            "unified-diff line numbers are 1-based"
-        )
-        assert self.body.startswith("@@"), (
-            "DiffHunk.body must start with the @@ hunk header"
-        )
+        if not self.file_path:
+            raise DiffParseError("DiffHunk.file_path must be non-empty")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)